src/Microsoft.AspNetCore.BuildTools.ApiCheck/Loader/AssemblyLoader.cs

```diff
@@ -1,33 +1,30 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Linq;
+#if NET452
+using System.IO;
+#endif
 using System.Reflection;
 #if NETCOREAPP1_1
-using NuGet.Frameworks;
 using NuGet.ProjectModel;
-#endif
 using NugetReferenceResolver;
-using System.IO;
+#endif
 
 namespace ApiCheck
 {
     public abstract class AssemblyLoader
     {
-        public static Assembly LoadAssembly(
-                string assemblyPath,
-                string assetsJson,
-                string framework)
+        public static Assembly LoadAssembly(string assemblyPath, string assetsJson)
         {
-            
 #if NETCOREAPP1_1
             var lockFile = new LockFileFormat().Read(assetsJson);
-            var graph = PackageGraph.Create(lockFile, framework);
+            var graph = PackageGraph.Create(lockFile, "netcoreapp1.1");
```

Comment:
As we discussed offline, moving to `netcoreapp2.0` hit some roadblocks. Would likely need to build this tool against a specific shared library version to ensure consuming projects run it in something newer. For now, leaving this as-is because it's earlier than what KoreBuild installs.

On the other hand, I have preserved the dependency version cleanup I did to get rid of package downgrades after moving every `.exe` to `netcoreapp2.0`.